README.md

```diff
@@ -1181,6 +1181,7 @@ This template was inspired by:
 
 Useful repositories:
 
+- [jxpress/lightning-hydra-template-vertex-ai](https://github.com/jxpress/lightning-hydra-template-vertex-ai) - Integration of Vertex AI custom training job and hyperparameter tuning job
```

Comment:
```suggestion
- [jxpress/lightning-hydra-template-vertex-ai](https://github.com/jxpress/lightning-hydra-template-vertex-ai) - integration of lightning-hydra-template with Vertex AI hyperparameter tuning and custom training job
```

Reply:
Just to make it more clear

Reply:
thank you for the suggestion
I have made the changes you suggested.